src/Pages/Home/Home.jsx

```diff
@@ -35,6 +35,16 @@ export const Home = () => {
                   {categoryStatus==="error" && <div>{categoryData}</div>}
             </div> 
 
+            <div className="spacer-5"></div>
+
+            {/* {brands} */}
+            <div class="flex flex-column flex-gap-2">
+                <h3 className="flex flex-justify-center category-header">Featured Brands</h3>               
+                    {brandStatus==="loading" && <div>Loading...</div>}
+                    {brandStatus==="success" && <BrandCard brandData={brandData} />}
+                    {brandStatus==="error" && <div>{brandData}</div>}
+            </div>
+
             <div className="spacer-5"></div>
```

Comment:
Can you explain what is this empty div for?

Reply:
It is for even spacing. It has height of 5rem. I have used it instead of increasing the margin of other elements to provide proper spacing between the elements.